README.md

```diff
@@ -272,79 +255,87 @@ And what is this ownership worth?
   called an **exit** or **liquidity event**. Sales, dissolutions, and bankruptcy are all called
   **liquidations**.
-- **Private sales:**
-  Liquidity events *may never happen*-if, say, the company stays private or shuts down.
-  But lacking the eventuality of a liquidity event, individuals may still buy or sell stock
-  (“gain liquidity”) in private companies in specific ways:
-  - 🄳🔹 In a few cases, you may be able to sell private company stock to another private
-    party, such as an accredited investor who wants to become an investor in the company.
-    This is called a **secondary market** transaction.
-    While many who hold stock hope or expect they need only find a willing buyer, in practice,
-    secondary sales only work out in a few situations.
+  - 🚧 “Becomes public” isn’t a clear enough definition of IPO. More detailed explanation of
+    “going public” top of this section at “public companies” defining occurrence, above.
+- **Private sales**: Liquidity events *may never happen*-if, say, the company stays private or
+  shuts down. But lacking the eventuality of a liquidity event, individuals may still buy or
+  sell stock (“gain liquidity”) in private companies in specific ways:
+  - 🄳 In a few cases, you may be able to sell private company stock to another private party,
+    such as an accredited investor who wants to invest in the company.
+    Sales between private parties like this are also called **secondary market** transactions,
+    to [distinguish them](https://www.investopedia.com/terms/s/secondarymarket.asp) from **primary
+    market** transactions where companies directly sell to investors.
+  - 🔸 While many who hold stock hope or expect they need only find a willing buyer, in
+    practice, secondary sales only work out in a few situations.
     In some cases, you may have luck selling stock privately to an individual, like a board
     member or former executive who wishes to increase their ownership.
-    Quora has
-    [more discussion](https://www.quora.com/How-do-employees-in-startups-sell-stock-in-the-secondary-markets)
-    on this topic.
+    You can find further discussion of this topic
+    [on Quora](https://www.quora.com/How-do-employees-in-startups-sell-stock-in-the-secondary-markets).
   - 🔸 Sales generally require the agreement and cooperation of the company, for both
     contractual and practical reasons.
-    - 🄳 Typically shares held are subject to a **right of first refusal** in favor of the
-      company, meaning someone can’t sell their shares to a third party without offering to sell
-      it to the company first.
-  - **Why companies may not support secondary sales:**
-    It’s important to consider a company’s perspective on secondary sales as well:
+    - 🄳 Typically, shares held by an employee are subject to a **right of first refusal** in
+      favor of the company, meaning someone can’t sell their shares to a third party without
+      offering to sell to the company first.
```

Comment:
Awkward phrasing WRT what is being sold. Too easy to mistake "offering to sell" with a company being sold. I propose an alternate for clarity:
"Typically, shares held by an employee are subject to a **right of first refusal** in favor of the company, meaning the employee can't sell their shares to a third party without offering to sell their shares to the company first."